metrics/wakumetrics/metrics.go

```diff
@@ -0,0 +1,141 @@
+package wakumetrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var (
+	MessagesSentTotal = prometheus.NewHistogramVec(
```

Comment:
Why is SomethingSomething**Total** a histogram? Shouldn't it be just a normal counter? How do you plan to use it as histogram?

Reply:
Updated all metrics to use gauge/counter as appropriate